tests/cephadm/test_admin_add_labels.py

```diff
@@ -0,0 +1,64 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def _add_host_label(instance):
+    """
+    Method add multiple labels to host
+    Args:
+        instance: cluster instance
+    Returns:
+        boolean
```

Comment:
Said returning `boolean` but returning integers in the code, Please correct that?

Reply:
Thanks @sunilangadi2
Updated